libs/wingsdk/test/http/http.test.ts

```diff
@@ -0,0 +1,145 @@
+import { test, describe, beforeEach, vi, Mock, expect } from "vitest";
+import {
+  Util as Http,
+  HttpMethods,
+  Response,
+  RequestCache,
+  RequestRedirect,
+} from "../../src/http/http";
+
+let defaultOptions = {
+  method: HttpMethods.GET,
+  headers: {},
```

Comment:
Can we please write this test with wing instead of typescript?

Reply:
yes, I'll add a wing test :) 
 also, since I'll have to use a real API resource- those can cover also the api's SDK spec tests! #2731